pkg/util/util.go

```diff
@@ -33,3 +36,65 @@ func GetRequestHost(req *http.Request) string {
 	}
 	return host
 }
+
+// RefreshValue contains a composite cookie refresh value setting
+type RefreshValue struct {
```

Comment:
Nice! I like this concept

Reply:
I think the organization is out of place.

This might fit best under `pkg/apis/middleware/refresh.go` -- at least under `apis/middleware` in one of the existing files if not its own.

Reply:
NIT: `RefreshOption` or `RefreshConfig` might be more clear that this manages something related to configuration.

Reply:
Moving to `middleware/refresh`, renaming as `RefreshOption`.

The reason for "Value" was because of the attempt to integrate with `pflag` custom *values*, which didn't work out due to wrong assumptions in `viper`